rir/R/rir.R

```diff
@@ -16,7 +16,12 @@ rir.disassemble <- function(what) {
 
 # compiles given closure, or expression and returns the compiled version.
 rir.compile <- function(what) {
-    .Call("rir_compile", what)
+    # TODO: gnu-r compile function takes optional environment argument for expressions
+    # this just uses globalenv for now
```

Comment:
This was inspired by the gnur functions `cmpfun` and `compile`. The first one gets a closure `f` and compiles it in a new compiler environment that is created from the `environment(f)`. The second gets an expression and an environment (that defaults to `.GlobalEnv`) and compiles with this. Since we only have one function as API, I tried to put all this in it..
Anyway, this was just in preparation to actually working with environments in the compiler, which is not yet the case.